packages/cardhost/app/templates/components/card-renderer.hbs

```diff
@@ -8,78 +8,83 @@ Ember Concurrency into the rendering of the card component here.
-            @buttonRoute="cards.card.view"
-            @buttonText="Done Editing"
-            @mode={{this.mode}}
-            @contextMenuOpen={{@contextMenuOpen}}
-            @setContextMenu={{@setContextMenu}}
-          />
-        {{/if}}
-        <div class="card-renderer-isolated--content">
-          <MaybeAnimate
-            @mode={{this.mode}}
-            @animationMode="view"
-            @content={{@card}}
-            @watch="isSelected"
-            @use={{this.borderAnimation}} as |card|>
-            <div class="card-renderer-isolated--card-container {{if (and this.cardstackSession.isAuthenticated card.isSelected) "selected"}}">
-              {{#if @dropField}}
-                <DropZone
-                  @dropField={{@dropField}}
-                  @position={{0}}
-                  @toggleStubField={{this.toggleStubField}}
-                />
-              {{/if}}
-              <Isolated
-                @card={{card}}
-                @fields={{this.fields}}
+      <AnimatedContainer>
+        <section
+          class="card-renderer-isolated
+            {{this.mode}}
+            {{safe-css-string @card.adoptedFromName}}
+            {{safe-css-string @card.name}}
+            {{if (and this.cardstackSession.isAuthenticated @cardSelected) "selected"}}
+            {{concat "field-count-" this.fields.length}}
+            {{@class}}
+          "
+          tabindex="0"
+          data-test-card-renderer-isolated
+          data-test-card-is-dirty={{if @card.isDirty "yes" "no"}}
+          {{on "focus" (action this.cardIsFocused true)}}
+          {{did-insert this.focusCard true}}
+          {{did-update this.updateFields @card.isolatedFields}}
+        >
+          {{#if (eq @mode "view")}}
+            {{#animated-value @card watch="isSelected" use=this.headerAnimation group="header"}}
+              <CardRendererHeader
+                @cardSelected={{if (and this.cardstackSession.isAuthenticated @cardSelected) true}}
+                @card={{@card}}
+                @buttonRoute="cards.card.edit.fields"
+                @buttonText="Edit"
                 @mode={{this.mode}}
-                @setFieldValue={{@setFieldValue}}
-                @setFieldName={{@setFieldName}}
-                @removeField={{@removeField}}
-                @toggleStubField={{this.toggleStubField}}
-                @setNeededWhenEmbedded={{@setNeededWhenEmbedded}}
-                @setPosition={{@setPosition}}
-                @selectField={{@selectField}}
-                @selectedField={{@selectedField}}
-                @dropField={{@dropField}}
+                @contextMenuOpen={{@contextMenuOpen}}
+                @setContextMenu={{@setContextMenu}}
               />
-            </div>
-          </MaybeAnimate>
-        </div>
-      </section>
+            {{/animated-value}}
+          {{else}}
+            <AnimatedContainer>
+              {{#animated-value @card watch=this.mode use=this.headerAnimation group="header"}}
```

Comment:
@burieberry `watch=this.mode` does actually do the thing you think it does. you can only watch properties on the item being animated, `@card` in this case. the `watch` field actually indicates the property on `@card` that you want to use to trigger the animation. because you are evaluating `this.mode` the `@card`property that you will actually end up watching is `@card.view` or `@card.edit` or `@card.schema`, etc.. which are nonsensical. `this.mode` will not trigger the animation.